README.md

```diff
@@ -26,89 +76,277 @@ LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](h
 - **unite_imu_method** parameter is now changeable in runtime.
 - **enable_sync** parameter is now changeable in runtime.
 
-
-
-## Installation Instructions
-
-   ### Step 1: Install the ROS2 distribution
- - #### Ubuntu 22.04:
-   - [ROS2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
- - #### Ubuntu 20.04: 
-   - [ROS2 Foxy](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html)
-   - [ROS2 Galactic](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)
- - #### Ubuntu 18.04 : 
-   - [ROS2 Dashing](https://docs.ros.org/en/dashing/Installation/Ubuntu-Install-Debians.html)
-   - [ROS2 Eloquent](https://docs.ros.org/en/eloquent/Installation/Linux-Install-Debians.html)
-
-
-### Step 2: Install the latest Intel&reg; RealSense&trade; SDK 2.0
-
-- #### Option 1: Install librealsense2 debian package (Not supported in Ubuntu 22.04 yet)
-   - Jetson users - use the [Jetson Installation Guide](https://github.com/IntelRealSense/librealsense/blob/master/doc/installation_jetson.md)
-   - Otherwise, install from [Linux Debian Installation Guide](https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md#installing-the-packages)
-      - In this case treat yourself as a developer: make sure to follow the instructions to also install librealsense2-dev and librealsense2-dkms packages
-
-- #### Option 2: Build from source
-  - Download the latest [Intel&reg; RealSense&trade; SDK 2.0](https://github.com/IntelRealSense/librealsense/releases/tag/v2.51.1)
+</details>
+    
+
+<h2 id="installation">
+  Installation
+</h2>
+  
+<details>
+  <summary>
+    Step 1: Install the ROS2 distribution 
+  </summary>
+  
+- #### Ubuntu 22.04:
+  - [ROS2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)
+- #### Ubuntu 20.04: 
+  - [ROS2 Foxy](https://docs.ros.org/en/foxy/Installation/Ubuntu-Install-Debians.html)
+  - [ROS2 Galactic](https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html)

```

Comment:
Let's add a `deprecated` notice on it (galactic)

Reply:
done